packages/reusable-blocks/src/store/actions.js

```diff
@@ -44,17 +44,22 @@ export const __experimentalConvertBlockToStatic =
  *
  * @param {string[]} clientIds The client IDs of the block to detach.
  * @param {string}   title     Reusable block title.
+ * @param {string}   blockType They type of block being created, reusable or pattern.
  */
 export const __experimentalConvertBlocksToReusable =
-	( clientIds, title ) =>
+	( clientIds, title, blockType ) =>
 	async ( { registry, dispatch } ) => {
+		let content = serialize(
+			registry.select( blockEditorStore ).getBlocksByClientId( clientIds )
+		);
+
+		if ( blockType === 'pattern' ) {
+			content = `<!-- wp:pattern -->${ content }<!-- /wp:pattern -->`;
```

Comment:
I'm not so sure about the idea of putting a pattern inside the reusable block CPT, as that'll be something that's hard to go back on (or migrate away from). Maybe some post meta or something could work?

The other thing to look at would be the ability to set a category, so both might entail changes to the CPT.

Reply:
Have switched to using post meta